assets/javascript/musicAPI.js

```diff
@@ -0,0 +1,25 @@
+
+
+
+
+
+// created function to get song by title making an ajax call to the indicated database
+function getSongs(songTitle){
+    // created url variable to call song from returned array from database also making an ajax call. 
+    var url = "https://musicdemons.com/api/v1/song";
+    $.ajax({
+        url: url, method:"GET"
+    }).then(function(response){
+        //Used filter method to compare selected song title queried from the URL with returned value in the response
+        var thisSong = response.filter(function(song){return song.title === songTitle})[0];
```

Comment:
is there an array that you are referencing with the '[0]' that we need to account for?